src/parser/parse_tree.h

```diff
@@ -3837,6 +3837,8 @@ typedef struct remote_cols
 typedef int (*PT_CASECMP_FUN) (const char *s1, const char *s2);
 typedef int (*PT_INT_FUNCTION) (PARSER_CONTEXT * c);
 
+#include "hide_password.h"
```

Comment:
Is there any reason for this include directives to place here instead of on the top of the file?

Reply:
Just mistake!
Thank you.